src/kudu/consensus/raft_consensus.cc

```diff
@@ -2534,13 +2534,26 @@ Status RaftConsensus::RequestVoteRespondVoteGranted(const VoteRequestPB* request
   MonoDelta backoff = LeaderElectionExpBackoffDeltaUnlocked();
   SnoozeFailureDetector(string("vote granted"), backoff);
 
+  // Get previous vote history here since it gets updated
```

Comment:
Wont having a reference to the map defeat the purpose of grabbing it before `SetVotedForCurrentTermUnlocked()`

Reply:
Good catch. I overlooked this,